### ISSUE TITLE
Seti UI light-theme color tokens not accessible

### ISSUE THREAD

**Issue by bgashler1** [microsoft/vscode#99187]
- VSCode Version: 1.46.0-insider (e1d8f5155c0db5ffdc130c9aa0f14efac02dfdfb)
- OS Version: macOS Catalina (10.15.3)

At least 7 of VS Code's [custom Seti-UI color tokens](https://github.com/microsoft/vscode/blob/1221df9e97d492093a41426d1fe188cda650e010/extensions/theme-seti/icons/vs-seti-icon-theme.json) are **below** 4.5:1 contrast ratio that's required to be visually accessible.

### Not passing contrast
`#498BA7` Blue
`#498ba7` Green
`#cc6d2e` Orange
`#dd4b78` Pink
`#9068b0`Purple 
`#b7b73b` Yellow

![Background colors Seti-UI icons sit on top of; the worst-case being the sidebar's active focus color, which is soft lavender that has little contrast against all existing colors used in VS Code Seti-UI icons, except red](https://user-images.githubusercontent.com/11839736/83571798-4fb1c700-a4dd-11ea-8bc6-301ac0447db0.png)

### Passing contrast
`#b8383d` Red ✅ (this is the only one that passes 4.5:1 contrast in all cases shown above)

## Steps to Reproduce:

1. To verify these findings, lookup the color values we're using for all "_light"-suffixed keys in the [vs-seti-icon-theme.json](https://github.com/microsoft/vscode/blob/1221df9e97d492093a41426d1fe188cda650e010/extensions/theme-seti/icons/vs-seti-icon-theme.json) file
2. Paste those values (or the ones I summarized above) into [WebAim's contrast checker](https://webaim.org/resources/contrastchecker/) as the foreground value over the background colors shown in the above image. The worst-case to ensure is passing is likely the sidebar's active selection background color.

Does this issue occur when all extensions are disabled?: Yes

## Recommended fix
Use darker colors that meet the minimum 4.5:1 contrast ratio for each background color shown above.

**Comment by bgashler1** [comment]
cc @misolori

Some may argue these icons are [purely decorative](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#puredecdef), thus not needing color-contrast per WCAG guidelines. However, these icons do serve a functional purpose of helping people visually parse which files are of what type, which is why I thought to report this as an issue.

**Comment by misolori** [comment]
@aeschli do you know if we're able to customize these colors or do they strictly come from the seti-ui theme?

**Comment by aeschli** [comment]
We take them unmodified from the Seti-theme: https://github.com/jesseweed/seti-ui/blob/master/styles/ui-variables.less. The same colors are used for light and dark.
Of course we could patch the colors when importing and also come up with different colors for light and dark.

**Comment by aeschli** [comment]
Closing as issue should be filed against https://github.com/jesseweed/seti-ui
